### ISSUE TITLE
Missing dependency for "aetest" package

### ISSUE THREAD

**Issue by extsoft** [CiscoTestAutomation/pyats#9]
If I install `aetest` with `pip install pyats.aetest`, I'll have the following error while executing a test:
```bash
$ python smoke_test.py 
Traceback (most recent call last):
  File "smoke_test.py", line 1, in <module>
    from ats import aetest
  File "..../versions/pyats/lib/python3.6/site-packages/ats/aetest/__init__.py", line 32, in <module>
    from .main import main
  File "src/ats/aetest/main.py", line 15, in init ats.aetest.main
  File "..../versions/pyats/lib/python3.6/site-packages/ats/aetest/loader/__init__.py", line 3, in <module>
    from .implementation import AEtestLoader
  File "src/ats/aetest/loader/implementation.py", line 7, in init ats.aetest.loader.implementation
  File "..../versions/pyats/lib/python3.6/site-packages/ats/aetest/reporter/__init__.py", line 4, in <module>
    from .aereport import AEReporter
  File "src/ats/aetest/reporter/aereport.py", line 2, in init ats.aetest.reporter.aereport
ModuleNotFoundError: No module named 'yaml'
```

File content starts with 
```python
from ats import aetest                                                                                                                                                        
from ats.aetest import setup, subsection, test, cleanup

class Instal(aetest.CommonSetup):
     @subsection
     def step_1(self):
         print('Download a ....')
```

**Comment by siming85** [comment]
will fix in next release

**Comment by siming85** [comment]
fixed in pyATS release v5.0.0, available as of Oct 9
